### PR TITLE
Added notes about supported Python versions and paths

### PR DESCRIPTION
**Pull request by owais**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ This Splunk distribution comes with the following defaults:
 
 > :construction: This project is currently in **BETA**.
 
+## Python3.5+ only
+
+This library does not support Python 2 and in fact only supports Python3.5+. If you still need Python 2 support,
+you should continue using signalfx-python-tracing.
```
Comment (brianashby-sfx) [inline README.md:21]:
```suggestion
## Requirements

This Splunk Distribution of OpenTelemetry requires Python 3.5 or later.
If you're still using Python 2, continue using the SignalFx Tracing Library
for Python.
```

File: README.md [hunk 2]
```diff
@@ -38,6 +43,21 @@ $ SPLK_SERVICE_NAME=my-python-app \
...
+  virtual environments, installing packages system-wide or in container images. In some cases, pip may install
+  packages into your user local environment. In that case you'll need to add your Python user base's bin directory
+  to your system path. You can find out your Python user base as follows by running `python -m site --user-base`.
+
+  For example, if `python -m site --user-base` reports that `/Users/splunk/.local` as the Python user base, then
+  you can add the directory to your path on Unix like system as follows:
+
+  ```
+  export PATH="/Users/splunk/.local/bin:$PATH"
+  ```
```
Comment (brianashby-sfx) [inline README.md:59]: can we move this to the requirements section?

**Comment by owais** [comment]
@brianashby-sfx can you form push this please so CI jobs get a chance to run again?

File: README.md (hunk 2 above)
Comment (owais) [reply README.md:71]: I don't think these belong in requirements. They're helpful generic python tips and not very specific to this library.

**Review by brianashby-sfx — APPROVED** [review]

**Pull request merged**